app/views/reservations/_new_reservation.html.erb

```diff
@@ -1,50 +1,57 @@
-<% title 'Create Reservation' %>
+<% title 'Confirm Reservation' %>
 
-<% unless @errors.empty? %>
-  <p>
-    <h3>
-      <i class="fa fa-exclamation-triangle warning-icon"></i>
-      Please be aware of the following errors:
-    </h3>
-  </p>
-  <% @errors.each do |msg| %>
-    <ul>
-      <li><%= msg %></li>
-    </ul>
+<div id="content" class="col-md-9">
+  <% unless @errors.empty? %>
+    <p>
+      <h3>
+        <i class="fa fa-exclamation-triangle warning-icon"></i>
+        Please be aware of the following errors:
+      </h3>
+    </p>
+    <div class ="form-errors">
+      <% @errors.each do |msg| %>
+        <ul>
+          <li><%= msg %></li>
+        </ul>
+      <% end %>
+    </div>
   <% end %>
-<% end %>
-
-<p>
-  <h3>Equipment being reserved:</h3>
-</p>
-
-<%= simple_form_for @reservation do |f| %>
   <p>
-    The following equipment will be reserved by
+    <h4>Your Reservation for
```

Comment:
Does this language apply to both people reserving for themselves as well as for others? The "Your" seems to imply it's only for reserving things for yourself; maybe we match the language in the request form and say "Equipment reserved for" or something like that.